server/pbench/bin/pbench-satellite-cleanup

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+import sys
+import os
+import shutil
+for line in sys.stdin:
+    tarball = line.split(": ")[1]
+    try:
+        src = "TO-SYNC/%s" % tarball
+        des = "SYNCED"
+        shutil.move(src, des)
+    except Exception as e:
+        print (e)
+        continue
+    try:
+        os.remove('%s' % tarball)
+    except Exception as e:
+        print (e)
+    try:
+        os.remove('%s.md5' % tarball)
+    except Exception as e:
+        print (e)
```

Comment:
@portante, @tenstormavi: This is an attempt to summarize where we are, where we should end up, and how to get there. I'm planning to keep the discussion here as a chain of replies, but if you think we should use some other method, let me know.

I realize that there is a problem here, but I would like to get this PR into the v0.43 release nevertheless. I propose that we take the smallest step possible: if there is a problem with ssh, then the script sends mail identifying the problem but does nothing else. If that possibility arises, then we will fix it manually.

I then propose that we fix the problem in the next release by following the design described in the [server scripts development documentation](http://distributed-system-analysis.github.io/pbench/doc/devel/server-scripts.html). Bireflly, that consists of splitting the rsync script into three pieces, one to do the rsync, one to process the tarballs and one to delete the synced tarballs, using state directories to keep track of what needs to be done, and retrying until successful. If there are problems, we will be notified, but once the problems are fixed, the next run of the scripts will take care of any delayed actions.